### ISSUE TITLE
Cant Pan in water

### ISSUE THREAD

**Issue by WhereIsMyDad** [anegostudios/VintageStory-Issues#1172]
v1.15.0
I tried panning and then the game crashed 
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.15.0 (Stable)7/16/2021 10:39:22 AM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.BlockPan.OnHeldInteractStart(ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, Boolean firstEvent, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel, EnumHandInteract useType, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsNoBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _DLw29xcc4gWHWfw8Em0aOjlQATi._8nnryhTwINaJcCBy90B7cqzCdfo(Single )
   at _Hx2yddcXbI4cHPSiFN8uLToeZYi._4nKLMGBXSYT0MlLMXytiZXnv1pE(Single )
   at _Hx2yddcXbI4cHPSiFN8uLToeZYi._zuHdBQu2Ufqe6gkW2Lbew2nkMld(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _aZ2fvA6N0dSuJcMaBGoJm3sgDNM._I7uMuBrykFfyAmYcxlmGjnH5B6N(_gOZk4eLYIxYBLj6wraBWI9aAKFz , String[] )
   at _Ftrf2yPR687y7LQA9DOeCuoEf4dA._I7uMuBrykFfyAmYcxlmGjnH5B6N(ThreadStart )
```

**Comment by Craluminum2413** [comment]
To reproduce this you need to pan bony soil

**Comment by copygirl** [comment]
In linked issue:
> just standing in the water, panning sand
> Crashed the game twice so far when i picked up the last sliver of sand with the pan

**Comment by radfast** [comment]
This issue is fixed in 1.15.1, released earlier this evening
